recycle-node.rb

```diff
@@ -80,19 +80,19 @@ def node_ready?(node)
 
 def worker_node?(node)
   node.dig("metadata", "labels")["kubernetes.io/role"] == "node" \
-    && node.dig("metadata", "labels", "kops.k8s.io/instancegroup") == WORKER_NODE_INSTANCEGROUP
+    && node.dig("metadata", "labels", "kops.k8s.io/instancegroup").include?(WORKER_NODE_INSTANCEGROUP)
 end
 
 def get_worker_instance_group_size
```

Comment:
We need to rename this method if it counts workers from multiple node groups. How about `target_worker_node_count` ?